api/reviews/index.js

```diff
@@ -0,0 +1,31 @@
+const express = require('express');
+
+const router = express.Router();
+
+const ProjectReviews = require('../../models/projectReviewsModel');
+
+router.post('/', async (req, res) => {
+    try {
+      const [projectReview] = await ProjectReviews.insert(req.body);
```

Comment:
**suggestion**: I don't see anywhere where you're validating the input to make sure ratings are between 1 and 5 stars. We can either do that at the database layer or just here in the route. What do you think?